documentation/book/assembly-storage.adoc

```diff
@@ -25,6 +25,13 @@ When configuring a `Kafka` resource, you can specify the type of storage used by
 
 The storage type is configured in the `type` field.
 
+.Additional resources
```

Comment:
What is the reason for moving this? I'm worried that:
* Additional resources suggests that this is the end. Can we remove that if we want to have the links in the middle of the assembly?
* We do not explain the types yet but tell the users where to go to read about the types. Wouldn't they be confused about the types? Shouldn't there be some quick introduction to the types before?

Also, can we do this in separate PR to make reviews easier?

Reply:
It seemed lost at the bottom of this section. Everything linked to is in that intro. Wouldn't you want to know about these types of storage before you actually looked at the procedures (or just skip the intro if you already do)?